fab/fabfile.py

```diff
@@ -32,7 +32,9 @@
 import sh
```

Comment:
@esoergel you may want to check this commit out. it changes some of the checks we did and how we used to get our tags/create out tags. i don't think the create tag will work quite yet, cause we'll need to authenticate the GitHub api object